source/making_payments/index.html.md.erb

```diff
@@ -9,6 +9,8 @@ weight: 2100
 
 This section tells you how to take a payment using the GOV.UK Pay API.
 
+If you want to take payments using a no-code solution instead, you can use [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/).
```

Comment:
what is the purpose of "instead" here? I am wondering if it works the same just with "If you want to take payments using a no-code solution, you can use..."

Reply:
Good point. Services can use both - I'll adjust.